kse/src/org/kse/gui/actions/GenerateDHParametersAction.java

```diff
@@ -0,0 +1,95 @@
+package org.kse.gui.actions;
+
+import java.awt.Toolkit;
+
+import java.util.ResourceBundle;
+
+import javax.swing.ImageIcon;
+import javax.swing.KeyStroke;
+
+
+import org.kse.gui.KseFrame;
+import org.kse.gui.dialogs.DGenerateDHParameters;
+import org.kse.gui.dialogs.DGeneratingDHParameters;
+import org.kse.gui.dialogs.DViewDHParameters;
+import org.kse.gui.error.DError;
+import org.kse.utilities.history.HistoryAction;
+
+
+public class GenerateDHParametersAction extends KeyStoreExplorerAction implements HistoryAction {
+	/**
+	 * 
+	 */
+	private static final long serialVersionUID = 7477452992392634450L;
+	protected static ResourceBundle res = ResourceBundle.getBundle("org/kse/gui/actions/resources");
+
+		public GenerateDHParametersAction(KseFrame kseFrame) {
```

Comment:
code quality: javadoc

Reply:
Updated.

Reply:
There is still no Javadoc for the constructor.

Also, the this line (32) is not correctly indented.

Reply:
Updated.